src/curve/stable/StableMintBurn.sol

```diff
@@ -119,9 +117,8 @@ contract StableMintBurn is ReservoirPair {
         _managerCallback();
     }
 
-    // TODO: public -> external?
     /// @dev Burns LP tokens sent to this contract. The router must ensure that the user gets sufficient output tokens.
-    function burn(address aTo) public returns (uint256 amount0, uint256 amount1) {
+    function burn(address aTo) external returns (uint256 amount0, uint256 amount1) {
```

Comment:
Do these changes need to be replicated to the main StablePair contract?

Reply:
yes good catch, 

fixed on the main `StablePair` contract